app/models/item.py

```diff
@@ -36,5 +37,6 @@ def to_dict(self):
             'createdAt': self.created_at,
             'updatedAt': self.updated_at,
             'seller': self.seller.to_dict(),
-            'category':self.category.name
+            'category': self.category.name,
+            'review_data': self.review_data.to_dict() if self.review_data else None
```

Comment:
Change `review_data` to `reviewData` since this is JSON we are creating. We should send a default dict instead of `None` so its easier on the front end. Set both as `0`.